src/com/company/entities/Sweet.java

```diff
@@ -0,0 +1,35 @@
+package com.company.entities;
+
+import com.company.entities.Product;
+
+public class Sweet extends Product {
+    boolean dietetic = false;
```

Comment:
redundant, default is false. it can be therefore just boolean dietetic;

Reply:
fixed